delfin/drivers/hitachi/hnas/nas_handler.py

```diff
@@ -0,0 +1,825 @@
+# Copyright 2021 The SODA Authors.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WarrayANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import hashlib
+import time
+
+import six
+
+from oslo_log import log as logging
+
+from delfin import exception, utils
+from delfin.common import constants
+from delfin.drivers.utils.ssh_client import SSHPool
+from delfin.drivers.hitachi.hnas import constants as constant
+from delfin.drivers.utils.tools import Tools
+
+LOG = logging.getLogger(__name__)
+
+
+class NasHandler(object):
+
+    def __init__(self, **kwargs):
+        self.ssh_pool = SSHPool(**kwargs)
+        self.evs_list = []
+
+    @staticmethod
+    def get_size(limit, is_calculate=False):
```

Comment:
Why the parameter of `get_size` is `limit`, should it be `size`?

Reply:
This function do the same thing as NetApp's get_size, so could put this part into `Tools.get_capacity_size`

Reply:
has deleted